test/integration-test/CMakeLists.txt

```diff
@@ -35,14 +37,15 @@ list(APPEND compiler_flags -std=c++11 -O2)
 list(APPEND library_flags ${Boost_LIBRARIES})
 
 list(APPEND gridtools_flags ${compiler_flags} ${include_flags} ${library_flags})
+
 gtclang_add_integrationtest(NAME no-codegen
                             GTCLANG ${GTCLANG_EXECUTABLE}
                             DIRECTORIES ${directories}
                             GRIDTOOLS_FLAGS ${gridtools_flags}
 )
 
 # Add a dummy target with all the source files so that they are included in a qt creator project
-dawn_add_custom_dummy_target(NAME qt_creator_parse_project_integration_test
+msbuild_add_custom_dummy_target(NAME qt_creator_parse_project_integration_test

```

Comment:
again see above for consistency of target naming.

Reply:
renamed as `GTClangQtCreatorParseProjectIntegrationTest`